getting-started/installation.md

```diff
@@ -97,6 +97,9 @@ If you have any problems here, check your `PATH`. `make install` will copy `wasm
 the case in general for building Go code from source.
 :::
 
+## jq
```

Comment:
This was pretty obvious to me, but worried it might confuse some less technical users as it does not come installed on macOS.